{"name": "test_sample_ui_test_case", "status": "broken", "statusDetails": {"message": "AttributeError: 'WebDriver' object has no attribute 'driver'", "trace": "self = <TestCases.testsuites.test_sample.TestSample1 object at 0x000001D948700A90>\nuiTestData = {'discharge_date': '01/18/2023', 'facility': {'facility_id': 'FA03', 'facility_name': 'Teamfacility'}, 'member': {'DOB...TN23', 'Member_id': 'TN22', ...}, 'password': '80f7d73c5456-430da38f0b7de6131f170c57755653164ab2a353a1cc138d12b9', ...}\nconfigLogger = <RootLogger root (INFO)>\n\n    def test_sample_ui_test_case(self,uiTestData, configLogger):\n        driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()))\n        driver.maximize_window()\n        login(uiTestData=uiTestData,driver=driver)\n>       login_web(uiTestData=uiTestData,driver=driver)\n\nTestCases\\testsuites\\test_sample.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nCommonClass\\web_utils\\CommonUiActions.py:44: in login_web\n    Member.createMember(driver)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"57ec54043cbc79896ee1e81df504502d\")>\n\n    def createMember(self):\n>       self.driver.find_element(By.XPATH, \"//*[@id='commandBarForm:searchButton']\").click()\nE       AttributeError: 'WebDriver' object has no attribute 'driver'\n\nCommonClass\\web_utils\\pages\\member.py:11: AttributeError"}, "steps": [{"name": "Log (INFO) ====== WebDriver manager ======", "status": "passed", "start": 1674126959841, "stop": 1674126959841}, {"name": "Log (INFO) Get LATEST chromedriver version for google-chrome 108.0.5359", "status": "passed", "start": 1674126963408, "stop": 1674126963408}, {"name": "Log (INFO) Driver [D:\\Users\\pvelu\\.wdm\\drivers\\chromedriver\\win32\\108.0.5359\\chromedriver.exe] found in cache", "status": "passed", "start": 1674126965100, "stop": 1674126965100}], "attachments": [{"name": "log", "source": "2b48c915-ccdd-4ae5-9353-87469f49999d-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "a588e4a9-d0ac-4a71-9d2d-ba2628778ffb-attachment.txt", "type": "text/plain"}], "start": 1674126959841, "stop": 1674126987353, "uuid": "ae3bbf7f-29c3-4031-8b01-94a5d4bbe8f5", "historyId": "1971dd38761290445127c4af03b141fd", "testCaseId": "2186558a34288d22382cd9d3e416f696", "fullName": "TestCases.testsuites.test_sample.TestSample1#test_sample_ui_test_case", "labels": [{"name": "parentSuite", "value": "TestCases.testsuites"}, {"name": "suite", "value": "test_sample"}, {"name": "subSuite", "value": "TestSample1"}, {"name": "host", "value": "WSAMZN-7K50OVEF"}, {"name": "thread", "value": "4792-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.testsuites.test_sample"}]}